src/core/gnsspositioninformation.h

```diff
@@ -233,6 +234,12 @@ class GnssPositionInformation
      */
     QString sourceName() const { return mSourceName; }
 
+    /**
+     * Returns the number of collected position from which the averaged positioning details were computed
+     * \note A value of zero means the position information isn't averaged
+     */
+    int averagedCount() const { return mAveragedCount; }
```

Comment:
We need to make these docs somehow user visible for the expression variables.
Certainly in the docs.
Ideally also in place in the qgis config, through qfieldsync, not sure if possible?